ftp.go

```diff
@@ -633,12 +633,6 @@ func (ftp *FTP) List(path string) (files []string, err error) {
 		if line, err = ftp.receiveNoDiscard(); err != nil {
 			return
 		}
-
-		if !strings.HasPrefix(line, StatusFileOK) {
```

Comment:
Instead of removing this check altogether, can we add the response we're seeing and consider it "success"?